cfme/middleware/deployment.py

```diff
@@ -0,0 +1,12 @@
+from cfme.fixtures import pytest_selenium as sel
+
+
+class Deployment(object):
+
+    def __init__(self, name):
+        self.name = name
+        self.provider = 'Middleware'
```

Comment:
Yes, then the usage in the navigation is like https://github.com/RedHatQE/cfme_tests/blob/master/cfme/containers/replicator.py#L14